app/controllers/revisions_controller.rb

```diff
@@ -7,15 +7,18 @@ class RevisionsController < AuthenticatedController
   before_action :load_record, except: [ :trash, :recover ]
 
   def index
-    redirect_to action: :show, id: @record.versions.where(event: 'update').last.try(:id) || 0
+    redirect_to action: :show, id: @record.versions.last.try(:id) || 0
```

Comment:
This changes, forces the `if` in L19, and when you're #show'ing a Revision that's not `update` then the `_empty` partial gets rendered giving the _false_ idea that there are no revisions for the record, there are (create, delete, restore), only that there are no _update_ revisions to show the diff.

I think the original behaviour was preferable, if you change the ID in the URL to some other random crap, you shouldn't get anywhere.